Zend/zend_operators.c

```diff
@@ -60,8 +60,8 @@ static _locale_t current_locale = NULL;
 /* Common code for SSE2 accelerated character case conversion */
 
 #define BLOCKCONV_INIT_RANGE(start, end) \
-	const __m128i blconv_start_minus_1 = _mm_set1_epi8((start) - 1); \
-	const __m128i blconv_end_plus_1 = _mm_set1_epi8((end) + 1);
+	const __m128i blconv_offset = _mm_set1_epi8((signed char)(CHAR_MIN - start)); \
```

Comment:
Is this correct now after the changes? If char is unsigned then CHAR_MIN is 0. And then you'll be casting it to a signed char. Wouldn't this incorrectly wrap in that case? I think using -128 was the correct way?

Reply:
Good point, this should use `SCHAR_MIN`.

Reply:
Fixed, thank you.